src/NuGet.Core/NuGet.Frameworks/FrameworkReducer.cs

```diff
@@ -69,6 +69,24 @@ public NuGetFramework GetNearest(NuGetFramework framework, IEnumerable<NuGetFram
                 }
             }
 
+
+            if (nearest == null || nearest.IsAny)
+            {
+                // Try ATF
+                var atfFallbackFramework = framework as AssetTargetFallbackFramework;
```

Comment:
```
/// <summary>
    /// AssetTargetFallbackFramework only fallback when zero assets are selected. These do not 
    /// auto fallback during GetNearest as FallbackFramework would.
 /// </summary>
```

This is the description mentioned for `AssetTargetFallbackFramework` do we know why this is explicitly mentioned not to fallback during GetNearest?